web/packages/authentik/package.json

```diff
@@ -0,0 +1,140 @@
+{
+    "name": "@goauthentik/authentik",
+    "version": "0.0.0",
+    "private": true,
+    "license": "MIT",
+    "scripts": {
+        "extract-locales": "lit-localize extract",
+        "build-locales": "run-s build-locales:build",
+        "build-locales:build": "lit-localize build",
+        "build-locales:repair": "prettier --write ./src/locale-codes.ts",
+        "rollup:build": "cross-env NODE_OPTIONS='--max_old_space_size=4096' rollup -c ./rollup.config.mjs",
+        "rollup:build-proxy": "cross-env NODE_OPTIONS='--max_old_space_size=4096' rollup -c ./rollup.proxy.mjs",
+        "rollup:watch": "cross-env NODE_OPTIONS='--max_old_space_size=4096' rollup -c -w",
+        "#build": "run-s build-locales rollup:build",
```

Comment:
HACK: Prevent Lage from "seeing" this package's `build`, so it won't try to run it.